### ISSUE TITLE
Dark Theme as well as Light Theme is not applying properly as soon as we change the app theme.

### ISSUE THREAD

**Issue by SB2318** [metabrainz/listenbrainz-android#107]
https://tickets.metabrainz.org/browse/LB-1241

@akshaaatt can I work on this topic?

**Comment by 07jasjeet** [comment]
It does work for me, can you share a screen recording please?

**Comment by SB2318** [comment]
@07jasjeet okay, will share.😇

**Comment by SB2318** [comment]
### Light-Theme-To-Dark:
https://user-images.githubusercontent.com/87614560/227785113-2e161093-5492-4b11-ad54-0aea8a59eea3.mp4

### Dark-Theme-To-Light:

https://user-images.githubusercontent.com/87614560/227785312-0cb9bc4e-dbe0-4593-a53f-dd7a09ba5cd2.mp4

@07jasjeet  I have seen most of the code you have done regarding this feature. 
So I will discuss with you about this on the respective channel before starting the work or if I face any difficulty.

**Comment by 07jasjeet** [comment]
Thanks for pointing this out! This only seems to be happening when internet connection is off. Weird compose bug. To start off, I would  recommend changing the mutable state variable to a class and add getter setter functions for it.

**Comment by SB2318** [comment]
> Thanks for pointing this out! This only seems to be happening when internet connection is off. Weird compose bug. To start off, I would recommend changing the mutable state variable to a class and add getter setter functions for it.

Yes, this is composed bug, currently, I am studying on this. And I have another plan also (If it is not possible to solve it). But before this, let me try. I will let you know both (you and @akshaaatt ) on the respective channel.

**Comment by akshaaatt** [comment]
Fixed.
